packages/store/mud.config.ts

```diff
@@ -0,0 +1,43 @@
+import { MUDConfig, ContractSchemaType } from "@latticexyz/cli";
+
+const config: MUDConfig = {
+  storePath: "../",
```

Comment:
what does `storePath` refer to?

Reply:
`import { StoreSwitch } from "../StoreSwitch.sol";`
By default storePath is `@latticexyz/store/src/`, but can be overridden
I should probably rename it, added it quickly just to make imports work